frontend/views/containers/settings/UserProfile.vue

```diff
@@ -170,9 +172,19 @@ export default {
       return false
     },
     async save () {
+      if (this.ephemeral.newPicture) {
```

Comment:
There are a few issues I can see happening with the way this `newPicture` variable is being used:

- What happens if there is an error? How does the user recover? Are they now forever prevented from editing any other part of their profile (requiring a refresh to reset)?
- Let's say the picture was uploaded successfully, and then they decide to change their display name. Well, the way it's coded the picture will be uploaded twice, and if it's the same picture it's possible this will result in the server returning an error (unclear how that's handled atm). It shouldn't re-upload the same picture twice.

Reply:
> What happens if there is an error?

They try to upload it again, there's nothing preventing them from it I believe.

> Let's say the picture was uploaded successfully, [...] Well, the way it's coded the picture will be uploaded twice.

Fixed by setting `newPicture` to false when the upload is done with success. That way it woun't upload again.